person.rb

```diff
@@ -0,0 +1,26 @@
+# Defines the Person class
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    if of_age? && @parent_permission
+      true
+    else
+      false
+    end
```

Comment:
- [OPTIONAL] This is good :+1:  But from my perspective, simplicity makes ruby a gem :gem: . The following code can be refactored to a single line and clearer.

See example:
```ruby
of_age?() || @parent_permission
```